codenames-domain/src/main/java/com/codenames/services/WordsService.java

```diff
@@ -0,0 +1,89 @@
+package com.codenames.services;
+
+
+import com.codenames.enums.Color;
+import com.codenames.enums.Language;
+import com.codenames.models.forooms.Settings;
+import com.codenames.models.forooms.Word;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.function.BiConsumer;
+
+public class WordsService {
+
+    private WordsService() throws IllegalAccessException {
+        throw new IllegalAccessException("Service class");
+    }
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WordsService.class);
+
+    private static final String RESOURCES_PATH = "codenames-domain/src/main/resources/";
+
+    private static final String FILE_NAME = "game-words-";
```

Comment:
definitely should not be hardcoded as well

Reply:
transferred it to `application.yml`